### PR TITLE
implement result view & setup compose test

### PR DESCRIPTION
**Pull request by yeonns**
## Description
- Implement collector result view 
  - Now, it is fake data
  - If euphony supports recording file, it will replaced by real data
- Setup testing for jetpack compose
  - Add dependencies for compose testing
  - Add test code of collectorView

## Screenshot of ResultView
<image src="https://user-images.githubusercontent.com/10149398/180013458-f9b18787-e1f0-4d2b-aef2-d20bbf68c738.jpg" width=320 />

## Screenshot of Testing Result
스크린샷, 2022-07-20 23-56-32

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -96,9 +96,15 @@ dependencies {
...
+    implementation "androidx.room:room-runtime:$room_version"
+    kapt "androidx.room:room-compiler:$room_version"
+    implementation "androidx.room:room-ktx:$room_version"
+
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
     androidTestImplementation "androidx.compose.ui:ui-test-junit4:$compose_version"
     debugImplementation "androidx.compose.ui:ui-tooling:$compose_version"
+    debugImplementation "androidx.compose.ui:ui-test-manifest:$compose_version"
```
Comment (designe) [inline app/build.gradle:109]: compose를 위한 ui-test dependencie 확인했습니다 :)

File: app/src/androidTest/java/co/jbear/euphony_hub/ui/view/CollectorViewTest.kt [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    @Test
+    fun assertThatPauseButtonIsDisplayed() {
+        composeTestRule.onNode(hasTestTag("textToSend"), useUnmergedTree = true)
+            .performTextInput("Hello")
+        composeTestRule.onNode(hasTestTag("speakerButton"), useUnmergedTree = true)
+            .performClick()
+        composeTestRule.onNode(hasTestTag("pauseButton"), useUnmergedTree = true)
+            .assertIsDisplayed()
+    }
+}
```
Comment (designe) [inline app/src/androidTest/java/co/jbear/euphony_hub/ui/view/CollectorViewTest.kt:44]: Compose에서의 UI unit-test 코드 확인하였습니다 👍

**Review by designe — COMMENTED** [review]
result view 구현 및 compose UI unit-test 셋업 감사합니다 :)
이름이 이름이 euphony_hub -> euphony_hub_collector로 바뀐만큼
package 명도 일괄 변경되면 좋을 것 같습니다!

`co.jbear.euphony_hub` -> `co.jbear.euphony_hub_collector`

**Comment by yeonns** [comment]
@designe Thank you for detailed review!
For package name, we have already issue https://github.com/euphony-io/euphony-hub-collector/issues/6, so how about applying it later? :eyes:

**Comment by designe** [comment]
> @designe Thank you for detailed review! For package name, we have already issue #6, so how about applying it later? eyes

I agree with you :+1:

**Review by designe — APPROVED** [review]

**Pull request merged**